definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1259,13 +1259,6 @@
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "uploadDocument",
-    "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
```

Comment:
See my last comment on the ticket. I think permissions should always at least be "R" otherwise the solicitor will not be able to see that those events have been triggered from the case history. To be confirmed by Jeremy I guess

Reply:
In addition to this, if you remove this here (or need to modify it to "R"), you need to add the original one to "CaseEvent-prod.json" so that production stays the same until our release.

Reply:
I agree with you.
I need confirmation from Jeremy but ive made the changes in antisipation  - https://github.com/hmcts/div-ccd-definitions/pull/634/commits/27e21ed73256e2f5a155d50e302d88438726677c